lib/summon/ris.rb

```diff
@@ -33,7 +33,7 @@ def ris_tags
 
       def ris_normal(value)
         if value.kind_of?(Array)
-          value.tag_per_value? ? value : [value.join(', ')]
+          value.tag_per_value? ? value : [value.join('; ')]
```

Comment:
I'm curious what's going on here. This looks like a change to the way the RIS library actually works?

Reply:
The join dividend in ris was changed from ,  to `;` cause `,` is the same as `,` between last  name and first name.

Reply:
I guess what I'm wondering is if we are making changes to the library itself, if there are any additional tests we need to do to verify that the changes work/are safe?

Reply:
Wait, I was wrong. The change was already in test branch on June 10. Not sure why it shows the difference. https://github.com/yalelibrary/summon-citation_export/blob/test/lib/summon/ris.rb
There is a test to verify the change https://github.com/yalelibrary/summon-citation_export/blob/rpsect_ris/spec/summon/ris_spec.rb#L38